community/modules/internal/slurm-gcp/instance_template/files/startup_sh_unlinted

```diff
@@ -23,6 +23,17 @@ if [[ -z "$HOME" ]]; then
 	HOME="$(getent passwd "$(whoami)" | cut -d: -f6)"
 fi
 
+# Temporary workaround for transition period when some of older images
+# don't have "baked in" python yet.
+# TODO: Remove after April 2025
```

Comment:
One month is not enough, what is impact of leaving it for a longer period of time?